### DISCUSSION TITLE
Running git:sync --build in background

### DISCUSSION THREAD

**Discussion by petrbroz** [dokku/dokku#5462] [Q&A]
Hi everyone,

I have a simple auto-deployment setup with a web server that handles webhooks from GitHub, and runs [git:sync --build](https://dokku.com/docs/deployment/methods/git/#initializing-from-private-repositories) to update the specific app via [dokku daemon](https://github.com/dokku/dokku-daemon).

However, since the dokku-daemon can only process one command at a time, and `git:sync --build` could potentially take a lot of time, I'm wondering if it's possible to somehow run the build in a non-blocking way (in the background)?

Thanks!

**Comment by josegonzalez** [comment]
There isn't any way to do what you're asking. Dokku Pro runs long-running processes in the background, and I expect the next version to have some level of git integration for running webhooks from github for deployment.
